tools/buildmgr/docs/cbuild.md

```diff
@@ -0,0 +1,74 @@
+# Build invocation with cbuild
```

Comment:
The files `tools/buildmgr/docs/cbuild.md` and `tools/buildmgr/README.md` are similar.